dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/UsersServiceImpl.java

```diff
@@ -610,14 +606,18 @@ public Map<String, Object> grantProjectByCode(final User loginUser, final int us
         }
 
         // 4. maintain the relationship between project and user
-        final Date today = new Date();
-        ProjectUser projectUser = new ProjectUser();
-        projectUser.setUserId(userId);
-        projectUser.setProjectId(project.getId());
-        projectUser.setPerm(7);
-        projectUser.setCreateTime(today);
-        projectUser.setUpdateTime(today);
-        this.projectUserMapper.insert(projectUser);
+        // check if already existed
```

Comment:
Is it better to use ```maintain the relationship between project and user if not exists``` to avoid misunstanding?

Reply:
we won't change the relationship when the relationship already exist. 
Do you mean that we should throw a error message to user?

Reply:
I mean, change the content of the comment. @leiwingqueen

Reply:
I see. I have change it